src/queries/hooks/rewards/usePostCreateActivity.ts

```diff
@@ -0,0 +1,9 @@
+import { useMutation } from '@tanstack/react-query';
+import { creatingActivity } from 'queries/queryKeys';
+import { createActivity } from 'queries/services/rewards';
+
+export const useCreateActivity = (payload: { userId: string; activityTypeId: string }) =>
```

Comment:
@abhishek-01k I think this should be claimRewardsActivity or completeRewardsActivity and not createActivity since activity is already there on backend, user is just claiming/completing them